src/Tracking.php

```diff
@@ -566,4 +566,61 @@ private static function filter_add_to_cart_attributes( array $args, WC_Product $
 
 		return $args;
 	}
+
+
+	/**
+	 * Get the formatted warning message for the potential conflicting tags.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return string The warning message.
+	 */
+	public static function get_third_party_tags_warning_message() {
+
+		$third_party_tags = self::get_third_party_installed_tags();
+
+		if ( empty( $third_party_tags ) ) {
+			return '';
+		}
+
+		return wp_kses_post(
+			sprintf(
+				/* Translators: %1$s Conflicting plugins */
```

Comment:
All of the numbered placeholders should be included here, not just the first one. Also, you can leave off the `%$s` portion, so it would look something like this:

```suggestion
				/* Translators: 1: Conflicting plugins, 2: Plugins Admin page opening tag, 3: Pinterest settings opening tag, 4: closing anchor tag */
```